rules-tests/Php81/Rector/Property/ReadOnlyPropertyRector/Fixture/withNamedConstructor.php.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Rector\Tests\Php81\Rector\Property\ReadOnlyPropertyRector\Fixture;
+
+class WithNamedConstructor
+{
+    private \DateTimeInterface $date;
+
+    public function __construct() {
+        $this->date = new \DateTime();
+    }
+    
+    public function getDate(): \DateTimeInterface
+    {
+        return $this->date;
+    }
+
+    public static function create(\DateTimeInterface $date): self
+    {
+        $ack = new self();
+        $ack->ackedAt = $date;
```

Comment:
This should same property name to make fixture valid:

```suggestion
        $ack->date = $date;
```

I will take care it ;)

Reply:
@Jean85  I cherry-picked your commit at PR https://github.com/rectorphp/rector-src/pull/4359